docs/pages/core_lib_doc/Data Layers/1.data_layers.md

```diff
@@ -45,7 +45,19 @@ class User(SoftDeleteMixin, Base):
     ...
 ```
 
-
+#### SQLAlchemy Structure:
+- <b>Imports:</b>
```

Comment:
before the import say a general comment that this is the common structure from sqlalchemy + add a link to the docs